ckanext/harvest/log.py

```diff
@@ -0,0 +1,16 @@
+from logging import Handler
+
+from ckanext.harvest.model import HarvestLog
+
+class DBLogHandler(Handler, object):
+    def __init__(self):
+        super(DBLogHandler,self).__init__()
+
+    def emit(self, record):
+        try:
+            level = record.levelname
+            msg = self.format(record)
+            obj = HarvestLog(level=level, content=msg)
+            obj.save()
+        except:
```

Comment:
*Never* do a blank `except` - this catches everything, including things like KeyboardInterrupt (such as someone hitting ^C). If you must catch all exceptions, you `except Exception`. Better would be to catch the specific exceptions you expect to see here, or at least log the exception.